### ISSUE TITLE
Default merge buttons

### ISSUE THREAD

**Issue by unional** [isaacs/github#747]
It is very nice that now github support both "merge commits" and "squash merging".

Able to disable one of them is great, but at times you would like to perform the disabled action.

I would be great if we can choose which is the default, or able or re-order them. 🌷

**Comment by notriddle** [comment]
If you only have squash merging enabled, merge commits can be done from the CLI.

If you only have merge commits enabled, you can squash commit using the CLI, but Github won't recognize it as a merge, which is kind of annoying.

I guess the simple answer is: if you want to be able to do it, don't disable it. :grinning:

**Comment by unional** [comment]
Yes, I would like to enable both of them. Just want to have the "squash merging" as the default. 🌷
